### PR TITLE
Update readme for community details

### PR DESCRIPTION
**Pull request by johnmcollier**
This PR updates the Codewind Readme to highlight how users can get in touch with us, as well as updates the Contributing section to add some more links. Based on Che's readme, whose organization I liked a lot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ Use the following instructions to install Codewind with your choice of editor:
...
 3. [Eclipse Che plugin](https://github.com/eclipse/codewind-che-plugin)
 
+## Feedback and Community
+
+Have any questions or commments on Codewind? Or would like to get involved in some way? Here's how you can get in touch with us:
+- **Support:** You can ask questions, report bugs, and request features using [GitHub issues](https://github.com/eclipse/codewind/issues).
+- **Public Chat:** Join the public [eclipse-codewind](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind) and [eclipse-codewind-dev](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind-dev) Mattermost channels
+- **Twitter:** [@EclipseCodewind](https://twitter.com/EclipseCodewind)
+- **Mailing List:** [codewind-dev@eclipse.org](https://accounts.eclipse.org/mailing-list/codewind-dev)
+- **Weekly Meetings:** We hold [weekly calls](https://github.com/eclipse/codewind/wiki/Codewind-Calls) every Tuesday and Thursday at 9:00 AM EST / 2:00 PM GMT.
```
Comment (jgwest) [inline README.md]: Minor suggestion - take it or leave it: `9:00 AM EST / 2:00 PM GMT.` -> `9:00AM Eastern Time / 14:00 UTC`

Comment (johnmcollier) [reply README.md]: I'm good with that

**Comment by johnmcollier** [comment]
@sishida Since I messing with the readme a bit, mind taking a look at my changes from an ID perspective? Thanks!

**Comment by sishida** [comment]
@johnmcollier I saw some little minor things that I'd like to change. I think it might be easiest if I approve and then create a new PR for my ID edits. Then, you could review my ID PR. Does that sound good? :)

**Comment by johnmcollier** [comment]
@sishida That sounds good to me!

**Comment by sishida** [comment]
@johnmcollier I have added my approval. Please feel free to merge, and I will then make a new PR. Thanks!

**Comment by jgwest** [comment]
@johnmcollier @sishida After seeing it on the repo front page, the new design, layout, and content look great 👍

**Pull request merged**